plugins/solidigm/solidigm-telemetry/data-area.c

```diff
@@ -302,6 +303,27 @@ static int telemetry_log_data_area_get_offset(const struct telemetry_log *tl,
 	return 0;
 }
 
+static int telemetry_log_nlog_parse(const struct telemetry_log *tl, struct json_object *formats,
+			     uint64_t nlog_file_offset,	uint64_t nlog_size,
```

Comment:
too many empty spaces

Reply:
I am assuming you meant to request to align argument to the parenthesis